### PR TITLE
Adding more repositories descriptions & alphabetical sorting

### PR DESCRIPTION
**Pull request by ttoine**
I sorted the list of repositories (keeping categories), so it will be more easy to add new ones or fix an existing one.

### CODE REVIEW

**Review by eternoendless — APPROVED** [review]
LGTM

File: coreteam.py [hunk 1]
```diff
@@ -47,37 +47,45 @@
...
     'ps_searchbar': 'Search Bar module',
-    'gsitemap': 'Google Sitemap module',
-    'ps_mainmenu': 'Main menu module',
     'ps_sharebuttons': 'Share Buttons module',
-    # misc
-    'classic-rocket': 'Classic-rocket theme',
+    'ps_shoppingcart': 'Shopping cart module',
+    'ps_socialfollow': 'Social Follow module',
+    'pscleaner': 'PS Cleaner module',
+    'statsregistrations': 'Registrations statistics modules'
```
Comment (ttoine) [inline coreteam.py]:
```suggestion
    'statsregistrations': 'Registrations statistics modules',
```

**Comment by ttoine** [comment]
I tested my small fix locally, it is working. thanks @PierreRambaud for noticing.

File: coreteam.py [hunk 2]
```diff
@@ -47,37 +47,45 @@
...
     'autoupgrade': 'Auto Upgrade module',
     'cronjobs': 'Cronjobs module',
+    'dashactivity': 'Dashboard Activity module',
+    'example-modules': 'Example modules',
+    'gamification': 'Gamification module',
+    'gsitemap': 'Google Sitemap module',
     'productcomments': 'Product Comments module',
+    'ps_contactinfo': 'Contact informations module',
+    'ps_emailalerts': 'Email Alerts module',
+     'ps_emailsubscription': 'Email subscription module',
```
Comment (Progi1984) [inline coreteam.py]:
```suggestion
    'ps_emailsubscription': 'Email subscription module',
```

File: coreteam.py [hunk 3]
```diff
@@ -47,37 +47,45 @@
...
-    'classic-rocket': 'Classic-rocket theme',
+    'ps_shoppingcart': 'Shopping cart module',
+    'ps_socialfollow': 'Social Follow module',
+    'pscleaner': 'PS Cleaner module',
+    'statsregistrations': 'Registrations statistics modules',
+  # misc
     'ADR': 'Architecture Decision Records repository',
+    'classic-rocket': 'Classic-rocket theme',
     'prestashop-specs': 'PrestaShop Specifications',
+    'traces': 'Traces',
```
Comment (Progi1984) [inline coreteam.py:88]: TopContributors ?

Comment (ttoine) [reply coreteam.py:88]: What do you think about that?
```suggestion
    'traces': 'Traces',
    'TopContributors': 'PrestaShop contributors website"
```

Comment (Progi1984) [reply coreteam.py:88]: ok for me

**Review by Progi1984 — APPROVED** [review]

File: coreteam.py [hunk 4]
```diff
@@ -47,37 +47,46 @@
...
+    'TopContributors': 'PrestaShop contributors website"
```
Comment (PierreRambaud) [inline coreteam.py]:
```suggestion
    'TopContributors': 'PrestaShop contributors website',
```

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Typo

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @ttoine

**Pull request merged**